release_tester/arangodb/instance.py

```diff
@@ -195,12 +195,32 @@ def launch_manual_from_instance_control_file(
         command = [str(sbin_dir / self.instance_string)] + self.instance_arguments[1:] + moreargs
         dos_old_install_prefix_fwd = str(old_install_prefix).replace("\\", "/")
         dos_new_install_prefix_fwd = str(new_install_prefix).replace("\\", "/")
+
+        # extract version from strings like that: 'arangodb3e-linux-3.10.2_x86_64'
+        version = re.search("(\d+\.\d+\.\d+)", new_install_prefix.name).group(1)
```

Comment:
I don't like this approach. It violates single source of truth. 

`cfg.version` etc. should be used, if its needed to be put in from the starterManager, we should rather do that, so the instance is aware of its expected version. This could as well then be used to check `Ready for Business [version]` log messages later on.

Reply:
In this function, value of `self.cfg.version` is not updated during upgrade. If I perform upgrade from 3.9.5 to 3.10.1, I will always see that version equals to 3.9.5 even if actual instance is 3.10.1. 

I tried to fix this version value but it seems that I need to dive more under hood to understand what's going on

Reply:
Fixed